docs/guides/liquidations.md

```diff
@@ -0,0 +1,84 @@
+# Liquidation Guide
```

Comment:
IMO this should be a section of the lending guide here: https://github.com/interlay/interlay.github.io/blob/master/docs/guides/lending.md